### PR TITLE
Feature/feature language catalan

### PR DESCRIPTION
**Pull request by xavier-campos-diaz**

### CODE REVIEW

**Review by enrichp — APPROVED** [review]
https://www.youtube.com/watch?v=USfx_JHMD3s

File: app/src/main/res/values-ca/strings.xml [hunk 1]
```diff
@@ -1,4 +1,89 @@
...
+    <string name="Recommended_kcal_random">747</string>
+    <string name="breed_random">Golden Retriever</string>
+    <string name="name_random">Firulais</string>
+    <string name="male">Mascle</string>
+    <string name="female">Femella</string>
+    <string name="age_random">4</string>
+    <string name="wash_random">30</string>
+    <string name="not_implemented">Aquesta funcionalitat encara no està implementada</string>
+    <string name="log_out">Tancar Sessió</string>
+    <string name="languages">Llenguatges</string>
```
Comment (AlbertPG) [inline app/src/main/res/values-ca/strings.xml]: You should put "Idiomes"

**Review by AlbertPG — COMMENTED** [review]
You should change that

**Review by AlbertPG — APPROVED** [review]
Okey

**Pull request merged**